src/gallery.js

```diff
@@ -40,35 +40,44 @@ const hideFullscreenGallery = ( galleryContainer ) => {
 
 	showFullscreenGallery = ( event, mediaItems, popup, doc = document ) => {
 		const fullscreenGallery = doc.createElement( 'div' ),
+			top = doc.createElement( 'div' ),
```

Comment:
do we need the top container that takes unnecessary 10vh space? 
The closes button element can stay alone without the container, because it has absolute position.

Reply:
discussion : Let's keep it and let design check in here.